### PR TITLE
added resources for exe - version information

### PR DESCRIPTION
**Pull request by sagarkchitnis**

### CODE REVIEW

File: src/vnsw/agent/contrail/SConscript [hunk 1]
```diff
@@ -104,8 +104,7 @@ if sys.platform.startswith('freebsd'):
 elif sys.platform.startswith('linux'):
     platform_dependent = 'linux/pkt0_interface.cc'
 elif sys.platform.startswith('win'):
-    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib']
-
+    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib', env.RES('windows/resource.rc')]
```
Comment (jacek-izykowski) [inline src/vnsw/agent/contrail/SConscript:107]: What's going to happen when someone modifies resource.h? Shall we add resource.h to platform_dependent files?

File: src/base/logging.cc [hunk 2]
```diff
@@ -61,9 +64,8 @@ void LoggingInit(const std::string &filename, long maxFileSize, int maxBackupInd
 
     if (useSyslog) {
         helpers::Properties props;
-		std::string syslogident;
-
-		//WINDOWS-TEMP syslogident = 	boost::str( boost::format("%1%[%2%]") % ident % getpid());
+        std::string syslogident;
+        syslogident = 	boost::str( boost::format("%1%[%2%]") % ident % osspecific_getpid());
```
Comment (jacek-izykowski) [inline src/base/logging.cc:68]: There's no `osspecific_getpid` for Linux.

**Review by jacek-izykowski — CHANGES_REQUESTED** [review]

File: src/vnsw/agent/contrail/SConscript (hunk 1 above)
Comment (sagarkchitnis) [reply src/vnsw/agent/contrail/SConscript:107]: It is in windows directory. Specific to windows only. Only if a new resource is added,someone will need to modify it.

File: src/base/logging.cc (hunk 2 above)
Comment (sagarkchitnis) [reply src/base/logging.cc:68]: ok. Will create a file with all these os_specific linux functions.

File: src/vnsw/agent/contrail/SConscript (hunk 1 above)
Comment (jacek-izykowski) [reply src/vnsw/agent/contrail/SConscript:107]: I understand that. But do we expect that modification of resource.h should trigger some action during next compilation?

Comment (sagarkchitnis) [reply src/vnsw/agent/contrail/SConscript:107]: Sorry I misunderstood your original question.  Yes, I just tried changing only resource.h. If you change resource.h, .rc is recompiled and the exe is re-linked.

File: src/base/logging.cc (hunk 2 above)
Comment (sagarkchitnis) [reply src/base/logging.cc:68]: put #ifdef's for linux instead of creating new file.

File: src/vnsw/agent/contrail/SConscript (hunk 1 above)
Comment (jacek-izykowski) [reply src/vnsw/agent/contrail/SConscript:107]: Ok. I just wanted to make sure that it works.

**Pull request closed without merging**